src/main/java/no/unit/nva/model/Typed.java

```diff
@@ -0,0 +1,14 @@
+package no.unit.nva.model;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeInfo.As;
+import com.fasterxml.jackson.annotation.JsonTypeInfo.Id;
+
+@JsonTypeInfo(use = Id.NAME, include = As.PROPERTY, property = "type")
+@JsonSubTypes({
```

Comment:
Db objects do not need to be annotated as the JSON serialization is handled by the DynamoDb Mapper